test/test_create_entities.py

```diff
@@ -0,0 +1,142 @@
+        self.scenario_runner = UnifiedSpecTestMixinV1()
+        spec = {
+            "description": "Find",
+            "schemaVersion": "1.2",
+            "createEntities": [
+                {
+                    "client": {
+                        "id": "client0",
+                        "uriOptions": {
+                            "retryReads": True
+                        },
+                    }
+                },
+                {
+                    "database": {
+                        "id": "database0",
+                        "client": "client0",
+                        "databaseName": "dat"
+                    }
+                },
+                {
+                    "collection": {
+                        "id": "collection0",
+                        "database": "database0",
+                        "collectionName": "dat"
+                    }
+                }
+            ],
+
+            "tests": [
+                {
+                    "description": "test loops",
+                    "operations": [
+                        {
+                            "name": "loop",
+                            "object": "testRunner",
+                            "arguments": {
+                                "storeIterationsAsEntity": "iterations",
+                                "storeSuccessesAsEntity": "successes",
+                                "storeFailuresAsEntity": "failures",
+                                "storeErrorsAsEntity": "errors",
+                                "numIterations": 5,
+                                "operations": [
+                                    {
+                                      "name": "insertOne",
+                                      "object": "collection0",
+                                      "arguments": {
+                                          "document": {
+                                              "_id": 1,
+                                              "x": 44
+                                          }
+                                      }
+
+                                    },
+                                    {
+                                        "name": "insertOne",
+                                        "object": "collection0",
+                                        "arguments": {
+                                            "document": {
+                                                "_id": 1,
+                                                "x": 44
+                                            }
+                                        }
+
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            ]
+        }
+
+        self.scenario_runner.TEST_SPEC = spec
+        self.scenario_runner.setUp()
+        self.scenario_runner.run_scenario(spec["tests"][0])
```

Comment:
This potentially could be written as:
```
   for test in spec["tests"]:
         self.scenario_runner.run_scenario(test)
```
This would be more general. However, if we know we aren't going to be adding on to the tests in this chunk of spec test, then I would prefer the first.

Reply:
Let's cross that bridge when we come to it. Glancing at  https://github.com/mongodb-labs/drivers-atlas-testing/tree/master/tests it appears that we only need to run a single test right now. 

Calling run_scenario twice will create a new entity_map so all the results from the previous test(s) will be lost. So I think the behavior of multiple tests is a little underspecified.